--- KisDlgAnimationRenderer.cpp	2023-03-30 09:30:05.208459196 -0700
+++ KisDlgAnimationRenderer2.cpp	2023-03-30 09:32:02.428268081 -0700
@@ -284,8 +284,18 @@
 
     m_page->ffmpegLocation->setStartDir(QFileInfo(m_doc->localFilePath()).path());
     KisConfig cfg(false);
-    QString ffmpegPath = cfg.ffmpegLocation();
-    QJsonObject ffmpegJsonObj = KisFFMpegWrapper::findFFMpeg(ffmpegPath.isEmpty() ? lastUsedOptions.ffmpegPath:ffmpegPath);
+    const QString cfgFFmpegPath = cfg.ffmpegLocation();
+    const QString likelyFFmpegPath = [&]() {
+        if (!cfgFFmpegPath.isEmpty()) {
+            return cfgFFmpegPath;
+        }
+
+        if (!lastUsedOptions.ffmpegPath.isEmpty()) {
+            return lastUsedOptions.ffmpegPath;
+        }
+
+        return QStandardPaths::findExecutable("ffmpeg");
+    }();
 
     ffmpegPath = ffmpegJsonObj["path"].toString();
     ffmpegVersion = ffmpegJsonObj["enabled"].toBool() ? ffmpegJsonObj["version"].toString():"None";
